packit/schema.py

```diff
@@ -171,6 +171,7 @@ class PackageConfigSchema(Schema):
     jobs = fields.Nested(JobConfigSchema, many=True, default=default_jobs)
     actions = ActionField(default={})
     create_pr = fields.Bool(default=True)
+    patch_generation_ignore_paths = fields.List(fields.String())
```

Comment:
Does it make sense to have `ignore_paths` for syncing files in general as well?

Reply:
I don't know to be honest. How would it work? Would synced_files had bigger prio than ignore_paths?

Reply:
Not sure, let's wait for some use-case...